packages/react/src/modules/core/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import "./styles.css";
+
+interface SwitchProps {
+  isChecked?: boolean;
+  onColor?: string;
+  onChange?: () => void;
+}
+
+export const Switch: React.FC<SwitchProps> = ({
+  isChecked = false,
+  onColor,
+  onChange,
+}) => {
+  console.log("is Checked", isChecked)
+  const [on, setOn] = useState(isChecked || false);
+
+  const toggleSwitch = () => {
+    if (onChange) {
+      onChange();
+      setOn(!on);
+    } else {
+      setOn(!on);
+    }
+  };
+
+  return (
+    <label className="switch">
```

Comment:
Wdyt about allowing customizing the label/input like by adding `name` or `id` for better accessibility? 

Alternatively, we could reach for something like Radix or Headless UI for high quality primitives out of the box.

Reply:
After talking with Krisna I'm removing the toggle entirely for now, I don't think it's the right time to use it